wasmdome/src/main.rs

```diff
@@ -30,18 +38,46 @@ enum WasmdomeAction {
     /// Configure local credentials to compete in the arena by submitting an access token
     Compete {
         /// Your account public key
+        #[structopt(short = "a", long = "account")]
         account: String,
 
         /// Short-lived access token granted by the wasmdome.dev website
+        #[structopt(short = "t", long = "token")]
         token: String,
     },
+    /// Query the schedule of upcoming matches
+    Schedule {},
+    /// Run a wasmdome match using the local lattice
+    Run {
+        /// Maximum number of turns in the match
+        #[structopt(short = "t", long = "max_turns")]
+        max_turns: u32,
+
+        /// Board height
+        #[structopt(short = "h", long = "height")]
+        board_height: u32,
+
+        /// Board width
+        #[structopt(short = "w", long = "width")]
+        board_width: u32,
+
+        /// Action points per turn
+        #[structopt(short = "p", long = "points")]
+        aps_per_turn: u32,
```

Comment:
I'm not really sure I want developers to be able to play with this, since the online version of the game will always be using 4. Maybe let's not include this as an option in the CLI and just use the default value?